From: Robin Lee <robinlee.sysu@gmail.com>
Date: Tue, 6 Dec 2016 17:26:45 +0800
Subject: Fix error on executing xdg-email.
Origin: https://bugs.launchpad.net/shutter/+bug/1469840
Bug: https://bugs.launchpad.net/shutter/+bug/1469840
Bug-Debian: https://bugs.debian.org/819149
Acked-by: Boyuan Yang <073plan@gmail.com>

This error occurs because shutter passed
"--attach file:///path/to/image" to xdg-email,
but --attach expects a file path, not a URI.
---
 bin/shutter | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/shutter b/bin/shutter
index 38eeedd..791057a 100755
--- a/bin/shutter
+++ b/bin/shutter
@@ -6967,7 +6967,7 @@ else {
             );
         }
         else {
-            push( @files_to_email, $session_screens{$key}->{'uri'}->to_string );
+            push( @files_to_email, $session_screens{$key}->{'long'} );
         }
 
         my $mail_string = undef;
